code/smart-mobility/smart-mobility.js

```diff
@@ -177,3 +177,129 @@ $(document).on("knack-scene-render.scene_281", function () {
 $(document).on("knack-scene-render.scene_284", function () {
   disableBreadCrumbsNonAdmin();
 });
+
+  //***MANAGE WALLET AND MAKE PAYMENT BUTTONS***
+$(document).on('knack-view-render.view_804', function(event, page) {
+  // create large button on go back to Portal page
+  bigButton('manage-wallet', 'view_804', "https://austin-tx-austin-tx.uat.cityba.se/welcome", "credit-card", "Manage Wallet");
+  // bigButton('manage-wallet', 'view_804', "https://atd.knack.com/smart-mobility#living-streets-applicant-portal/", "credit-card", "Manage Wallet");
+});
+$(document).on('knack-view-render.view_806', function(event, page) {
+  // create large button on go back to Portal page
+  // *** see the getCitybaseButton function for the invocation of the make payment bigButton function
+  //   bigButton('make-payment', 'view_806', "https://atd.knack.com/smart-mobility#living-streets-applicant-portal/", "arrow-right", "Make Payment");
+});
+
+/** 
+ * Citybase Integration
+ */
+
+function getCitybaseButton(payload, viewId) {
+  fetch(
+    "https://invoice-service-austin-tx.uat.cityba.se/invoices/austin_tx_transportation/street_banner",
+    {
+      method: "POST",
+      headers: {
+        "Content-Type": "application/json",
+      },
+      body: JSON.stringify(payload),
+    },
+  )
+    .then((response) => {
+      console.log(response.url);
```

Comment:
I'm not sure if it's important to keep the information being emitted onto the console useful beyond the developer at the time -- but, I think if we're going to leave in `console.log()`s, they should be at least a little self documenting, like 

```
console.log("Citybase response's URL:", response?.url);
```